mne/report/report.py

```diff
@@ -490,14 +490,17 @@ def _itv(function, fig, **kwargs):
          np.concatenate(images[3:], axis=1)],
         axis=0)
 
-    dists = dig_mri_distances(info=kwargs['info'],
-                              trans=kwargs['trans'],
-                              subject=kwargs['subject'],
-                              subjects_dir=kwargs['subjects_dir'])
-
+    try:
+        dists = dig_mri_distances(info=kwargs['info'],
+                                  trans=kwargs['trans'],
+                                  subject=kwargs['subject'],
+                                  subjects_dir=kwargs['subjects_dir'])
+        caption = (f'Average distance from {len(dists)} digitized points to '
+                   f'head: {1e3 * np.mean(dists):.2f} mm')
+    except BaseException as e:
+        caption = ('Distances could not be calculated from digitized points')
+        logger.warning(e)
```

Comment:
Instead of `logger.warning` this should be `warn(...)` from `..utils`. It does smart emission based on `verbose` level